script.js

```diff
@@ -209,4 +257,57 @@ const app = {
   }
 };
 
-app.run();
+
+const manageDom = function() {
+  let dom, fn, catchDom, addEvents, init
+  dom = {}
+  fn = {}
+
+  catchDom = function(){
+    dom.buttonShowForm = document.getElementById("js-button-show-form")
+    dom.buttonHideForm = document.getElementById("js-button-hide-form")
+    dom.blockForm = document.getElementById("js-block-form")
+    dom.buttonSendNewTask = document.getElementById("send-new-task")
+    dom.selectFilter = document.getElementById("js-select-filter")
+    dom.taskList = document.getElementById("js-task-list")
```

Comment:
I like how do you naming your variables, but I don't know if this a good form :thinking:

Reply:
Trust me. Believe me.